doandroidlib/src/main/java/in/tosc/doandroidlib/objects/NewDropletRequestBody.java

```diff
@@ -0,0 +1,133 @@
+package in.tosc.doandroidlib.objects;
```

Comment:
name of an API object should not be contain stuff like 'new' and 'body'
It will go in POST, so body is implied. It will be a new request, that is also implied becuase we are *creating* droplet.